Datadog.Trace.proj

```diff
@@ -131,6 +131,26 @@
     </MSBuild>
   </Target>
 
+  <Target Name="PublishBuildLoggerOnDisk">
+    <ItemGroup>
+      <BuildLoggerPublishProject Include="src\Datadog.Trace.MSBuild\Datadog.Trace.MSBuild.csproj">
+        <Properties>TargetFramework=net45;PublishDir=$(TracerHomeDirectory)\net45</Properties>
```

Comment:
Is this the right place for these files? Despite its name, `$(TracerHomeDirectory)` is only used for filed needed for automatic instrumentation. For example it's used to generate the msi installer, but not the nuger packages used to manual instrumentation.

I guess the real question is: how do we plan to distribute the MSBuild integration?

Reply:
If the answer is "it will be installed in the GAC using the msi installer", then ignore the comment above 😉

Reply:
But what about .NET Core?

Reply:
The current idea is to distribute the `xunit` and `msbuild` instrumentation as part of the current distribution of the tracer, because to make it work the user doesn't need to change the code, neither add new dependencies but it's done kind of automatic (in `msbuild` we have to manually add a logger in the command line but doesn't depend of the target project). For BenchmarkDotNet is quite different because the exporters are declared inside the code, so the user would have to add a new dependency, in that case make sense to have a nuget package.

For .NET Core I don't know what is the current distribution for the tracer in this case. But we have an option to add a dotnet global tool?? if we need it to make the dev life easier.

Reply:
Ok, you mean it will be distributed in the msi. That makes sense. Resolving this thread.